projects/components/src/filtering/filter-button/filter-button.component.ts

```diff
@@ -45,7 +45,7 @@ export class FilterButtonComponent implements OnChanges {
   public attribute?: FilterAttribute;
 
   @Input()
-  public value?: unknown;
+  public value?: FilterValue;
```

Comment:
Seems completely dependent on the filter implementation - what's the point of typing it?

Reply:
This is required for Filter to GqlFilter transformation

Reply:
This is the kind of thing I was saying the other day we shouldn't try to make generic, because there is no one GQL filter representation.